### ISSUE TITLE
Edited by on Projects

### ISSUE THREAD

**Issue by KyndallH** [ArctosDB/arctos#3682]
<img width="1959" alt="Screen Shot 2021-06-24 at 2 44 15 PM" src="https://user-images.githubusercontent.com/16887896/123341626-bf389100-d4fa-11eb-9baa-b84fa6a802d9.png">

The placement of the agent editing the record is poorly placed. It makes it look like they are more involved in the project than editing. Plus - joined and left??? It also looks like they have been working on this project from 2008 and their involvement is ongoing.

I suggest that editing agent information be placed at the bottom of the project record.

Also, I went to this project and deleted a source along with added a source but I was not recorded as having edited this record. Hmmm. 

Priority: Moderate. Sometime in the next two years.

**Comment by Jegelewicz** [comment]
I unassigned Mariel - this is something for Dusty.

**Comment by dustymc** [comment]
>bottom

Not sure splitting agents makes sense to me, but whatever - possibly best discussed as part of https://github.com/ArctosDB/arctos/issues/3468.

>joined and left??? I

https://github.com/ArctosDB/arctos/issues/3584 - it ain't my fault.

>added a source 

I'm not sure what that means, but the current trigger is on table project, that's it. That could be altered, needs discussed in a separate issue.

**Comment by Jegelewicz** [comment]
I agree that the editing agent stuff ends up looking weird. Here's a project on which I am an actual participant

![image](https://user-images.githubusercontent.com/5725767/128553580-360e0f5d-e0a3-45b2-96c9-1c8f614c8264.png)

I edited stuff today so now I have a join and leave date of today, but also my actual project join date which was way back when in 2018, although that isn't really clear here. Also, my edit overrides the more important role (research associate).

![image](https://user-images.githubusercontent.com/5725767/128553886-0120d4c3-5d87-4d7c-821c-eb054cc22939.png)

I do think it would be better if creating and editing were viewed differently from "participating" and last edited by should go at the very bottom of the project page.

**Comment by Jegelewicz** [comment]
I think @KyndallH question about joined and left??? relates to the editor. I don't join every project I happen to edit!

**Comment by dustymc** [comment]
>Here's a project

Links would be pretty handy...

**Comment by Jegelewicz** [comment]
HAHAHA! YES they would - sorry about that!

https://arctos.database.museum/project/10003185

**Comment by dustymc** [comment]
I think the form doesn't properly handle single agents in multiple project roles. It should.

**Comment by dustymc** [comment]
Moved project dates up, tweaked the CSS.

Moved admin agents into new CSS. (Note that this is only visible to "us.")

Confirmed that all agents are being displayed.

**Comment by Jegelewicz** [comment]
When I look at the project above in test, I no longer see the "edited by" part

http://test.arctos.database.museum/project/10003185

**Comment by Jegelewicz** [comment]
> When I look at the project above in test, I no longer see the "edited by" part
> 
> http://test.arctos.database.museum/project/10003185

It seems like something has been lost...

**Comment by dustymc** [comment]
It's test - the data are older than this functionality. Try editing it.
